docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -269,6 +270,39 @@ $ tctl edit cluster_auth_preference
 
 </TabItem>
 </Tabs>
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
```

Comment:
I would recommend using a two-dimensional `Tabs` component instead of a `ScopedBlock` here since users tend not to realize that you can adjust the scope of a docs page. We should really only use `ScopedBlock`s when the risk of a user seeing the wrong instructions with the wrong scope selected is minimal.

Reply:
Here's what I mean. This also lets you remove one of the `TabItems`, since you can reuse it:

````diff
diff --git a/docs/pages/access-controls/guides/device-trust.mdx b/docs/pages/access-controls/guides/device-trust.mdx
index f59bff53dd..4035499147 100644
--- a/docs/pages/access-controls/guides/device-trust.mdx
+++ b/docs/pages/access-controls/guides/device-trust.mdx
@@ -220,9 +220,8 @@ preview. Only `tsh` and Teleport Connect are able to fulfill device mode
 
 To enable device mode `required` update your configuration as follows:
 
-<ScopedBlock scope={["enterprise"]}>
-<Tabs>
-<TabItem label="Static Config">
+<Tabs dropDownCaption="Teleport Deployment">
+<TabItem label="Static Config" options="Self-Hosted">
 Edit the Auth Server's `teleport.yaml` file:
 
 ```diff
@@ -237,7 +236,8 @@ auth_service:
 ```
 
 </TabItem>
-<TabItem label="Dynamic Resources" >
+<TabItem label="Dynamic Resources" options="Self-Hosted,Teleport Enterprise Cloud">
+
 Create a `cap.yaml` file or get the existing configuration using
 `tctl get cluster_auth_preference`:
 
@@ -270,39 +270,6 @@ $ tctl edit cluster_auth_preference
 
 </TabItem>
 </Tabs>
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
-
-Create a `cap.yaml` file or get the existing configuration using
-`tctl get cluster_auth_preference`:
-
-```diff
-kind: cluster_auth_preference
-version: v2
-metadata:
-  name: cluster-auth-preference
-spec:
-  type: local
-  second_factor: "on"
-  webauthn:
-    rp_id: example.com
-  device_trust:
-+   mode: "required" # add this line
-```
-
-Update the configuration:
-
-```code
-$ tctl create -f cap.yaml
-cluster auth preference has been updated
-```
-
-You can also edit this configuration directly:
-
-```code
-$ tctl edit cluster_auth_preference
-```
-</ScopedBlock>
 
 SSH, Database and Kubernetes access without a trusted device is now forbidden.
 For example, SSH access without a trusted device fails with the following error:
````

Reply:
Thanks @ptgott , please see update.  I put the Dynamic resources first so this would show and since it works for all examples.